package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "dreame-backend",
+  "packageManager": "yarn@4.1.1",
+  "description": "Dreame Piedteam Backend",
+  "main": "./dist/index.js",
+  "repository": {
+    "type": "module"
+  },
+  "scripts": {
+    "server": "npx ts-node ./src/index.ts",
+    "dev": "yarn run nodemon",
+    "build": "rimraf ./dist && tsc && tsc-alias",
+    "start": "node dist/src/index.js --production",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "prettier": "prettier --check .",
+    "prettier:fix": "prettier --write ."
+  },
+  "author": "",
+  "license": "MIT",
```

Comment:
"author": "",
"license": "MIT"
remove these 2 and at this at the top
"private": true,